roles/openshift_node/tasks/apply_machine_config.yml

```diff
@@ -0,0 +1,115 @@
+---
+- name: Create temp directory
+  tempfile:
+    state: directory
+  register: temp_dir
+
+- name: Get cluster pull-secret
```

Comment:
Isn't pull secret already stored on the node, somewhere in `/var/lib/kubelet`?

Reply:
Confirmed `/var/lib/kubelet/config.json`.
I'll update to use that instead.